website/content/concepts/layer2.md

```diff
@@ -67,6 +67,31 @@ about 10s), please [file a bug](https://github.com/metallb/metallb/issues/new)!
 We can help you investigate and determine if the issue is with the client, or a
 bug in MetalLB.
 
+## How the L2 leader election works
+
+The election of the "leader" (the node which is going toadvertise the IP) of a
+given loadbalancer IP is stateless and works in the following way:
+
+- each speaker collects the list of the potential announcers of a given IP, taking
+into account active speakers, external traffic policy, active endpoints, node selectors and other things.
+- each speaker does the same computation: it gets a list of a hash of "node/VIP" elements and
+picks the first item of the list.
```

Comment:
picks the -> announces the service if it is the first item of the list

Reply:
done